.github/workflows/audit.yml

```
@@ -0,0 +1,41 @@
+name: Security audit
+
+on:
+  schedule:
+    # Runs at 00:00 UTC everyday
+    - cron: '0 0 * * *'
+  push:
+  pull_request:
+
+jobs:
+  rust-audit:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: EmbarkStudios/cargo-deny-action@3f4a782664881cf5725d0ffd23969fcce89fd868 # v1.6.3
+        with:
+          manifest-path: "deps/swc/bindings/Cargo.toml"
```

```suggestion
          manifest-path: "deps/swc/bindings/binding_typescript_wasm/Cargo.toml"
```

This would work.

If you want to run cargo deny on the SWC itself, you need to store `./crates` too.